src/constants/appConstant/url.ts

```diff
@@ -1,7 +1,7 @@
-//import {githubConfig} from '../../../config/config';
+import {githubConfig} from '../../../config/config';
 // you can include client id from readme file in config and import it as above.
 
-import {githubConfig} from '../../../config/configSample';
+// import {githubConfig} from '../../../config/configSample';
```

Comment:
Can you please remove unwanted commented code?

Reply:
will resolve in next commit .